src/main/scala/symsim/concrete/ConcreteExactRL.scala

```diff
@@ -33,3 +33,24 @@ trait ConcreteExactRL[State, ObservableState, Action]
   /** Convert the matrix Q after training into a Policy map. */
   def qToPolicy (q: Q) (using Ordering[Double]): Policy =
     q.states.map { s => (s, vf.bestAction (q) (s)) }.to (Map)
+
+  def policyEval (policies: List[Policy]): List[List[Double]] =
```

Comment:
Apologies, but I insist that the argument has to be a **single policy**. All this code with mapping does not belong to the implementation of the learning/evaluation algorithm, but to do the client code. It should be in Experiments.scala.

If you expect it will be reused in multiple models it should be placed in a similar place as the implementation of `learnAndLog`. It is a similar utitlity function. (We may later move these to a Util module, but for now placing it in ExpermentSpec seems convenient.)